app/src/controllers/filter/constants.js

```diff
@@ -26,7 +26,7 @@ export const DEFAULT_FILTER = {
   entities: [],
   type: 'launch',
   orders: [
-    { is_asc: false, sorting_column: 'start_time' },
-    { is_asc: false, sorting_column: 'number' },
+    { isAsc: false, sortingColumn: 'start_time' },
```

Comment:
maybe `startTime`?

Reply:
Yes, it should be another fix for Launch fields (in separate Pull Request)